example/ios/StripeSdkExample/Info.plist

```diff
@@ -60,6 +70,8 @@
 	<array>
 		<string>armv7</string>
 	</array>
+	<key>LSApplicationQueriesSchemes</key>
+	<string>weixin</string>
```

Comment:
Is `LSApplicationQueriesSchemes` only needed if we call `canOpenURL`? Since we're not doing that here, maybe we can omit?

Reply:
yes you're right